src/Materials/babylon.materialHelper.ts

```diff
@@ -103,6 +104,20 @@
                     }
                 }
 
+                if (light.lightmapMode != 0 ) {

```

Comment:
And this is where the naming will be important to help users understand each mode